### ISSUE TITLE
FATAL rendering error in quiz header

### ISSUE THREAD

**Issue by iainbryson** [dtu-compute/dtu-quiz#233]
Noticed this while looking through the logs at other things:

```
D, [2017-02-07T14:34:21.823543 #772] DEBUG -- : Groups for s123112 [
    ^[[0;37;49m[0] ^[[0m#<DTUAuth2::UserRole2:0x007fdbfda2b1c0 @group="Main", @user_id="s123112", @course_id="02402", @acl="author">
]
D, [2017-02-07T14:34:21.823860 #772] DEBUG -- : Courses for s123112 [
    ^[[0;37;49m[0] ^[[0m^[[0;33m"02402"^[[0m
]
I, [2017-02-07T14:34:21.824470 #772]  INFO -- : Course list for s123112 [
    ^[[0;37;49m[0] ^[[0m{
           :course_id^[[0;37m => ^[[0m^[[0;33m"02402"^[[0m,
                 :url^[[0;37m => ^[[0m^[[0;33m"/courses/02402/admin"^[[0m,
               :title^[[0;37m => ^[[0m^[[0;33m"02402 (Author)"^[[0m,
                 :acl^[[0;37m => ^[[0m^[[0;33m"author"^[[0m,
        :users_groups^[[0;37m => ^[[0m[
            ^[[0;37;49m[0] ^[[0m#<DTUAuth2::UserRole2:0x007fdbfda2b1c0 @group="Main", @user_id="s123112", @course_id="02402", @acl="author">
        ]
    }
]
I, [2017-02-07T14:34:21.843412 #772]  INFO -- : Active quizzes 48|60|62|65
I, [2017-02-07T14:34:21.908502 #772]  INFO -- : got checksum c73b208d28d5d794f23a4aa07cb82a62
I, [2017-02-07T14:34:21.910081 #772]  INFO -- :   Rendered shared/_user_nav.html.erb (0.2ms)
I, [2017-02-07T14:34:21.911022 #772]  INFO -- :   Rendered shared/_course_header.html.erb (1.6ms)
I, [2017-02-07T14:34:21.911171 #772]  INFO -- :   Rendered student_quizzes/index.html.erb within layouts/application (1.9ms)
I, [2017-02-07T14:34:21.911325 #772]  INFO -- : Completed 500 Internal Server Error in 237ms
F, [2017-02-07T14:34:21.912660 #772] FATAL -- :
ActionView::Template::Error (undefined method `[]' for nil:NilClass):
    31:   <div class="col-md-6 col-md-offset-3">
    32:   <select class="form-control selectpicker" onchange="location = this.options[this.selectedIndex].value;">
    33:     <% if course_user_options.current_course && %>
    34:       <option value="" disabled selected><%= course_user_options.current_course.select{ |o| o[:acl] == current_acl }.first[:title] if current_acl %></option>
    35:       <% for course in course_user_options.other_courses_list do %>
    36:       <option value="<%= course[:url] %>"> <%= course[:title] %></option>
    37:       <% end %>
  app/views/shared/_course_header.html.erb:34:in `_app_views_shared__course_header_html_erb___2393913788431859414_70291415921260'
  app/views/student_quizzes/index.html.erb:20:in `_app_views_student_quizzes_index_html_erb___89633026148309733_70291415787480'
  app/controllers/student_quizzes_controller.rb:22:in `block (2 levels) in index'
  app/controllers/student_quizzes_controller.rb:21:in `index'

```

**Comment by dtu-compute** [comment]
I am looking forward to hear what that could explain of the issues we are seeing now =)

**Comment by iainbryson** [comment]
Possibly it’s related, but it seems to happen for other users too ¯\_(ツ)_/¯

**Comment by iainbryson** [comment]
resolved by commit 53e935c8f914b23fb5a6264b85aa61fcb8871266
